staging/src/k8s.io/controller-manager/controller/dummy.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import "net/http"
+
+var _ Interface = (*DummyController)(nil)
+var _ Debuggable = (*DummyController)(nil)
+
+// DummyController is a controller that does nothing
```

Comment:
Can we elaborate on the intended usage? Is this mean for tests? (In which case we may want to convey that in the path).  We don't seem to be using this currently?

Reply:
remove, same as the split PR